spec/features/view_pages_spec.rb

```diff
@@ -6,4 +6,10 @@
 
     expect(page).to have_text("Lorem")
   end
+
```

Comment:
We definitely want request specs rather than feature specs. They are a pain. Let the QA guy do the feature specs as they will just slow us down. They tend to be very brittle and time consuming.

Reply:
Thanks, request tests look great!